service/service_test.go

```diff
@@ -0,0 +1,253 @@
+package service
```

Comment:
each student and teacher should have separate test class. `student_test` and `teacher_test`